scripts/us_bea/states_gdp/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "USStatesQuarterlyGDP",
+            "curator_emails": [
+                "fpernice@google.com"
```

Comment:
\# TODO(jeffreyoldham): Update the curator when appropriate.

Reply:
added to the README